mlkem/src/native/meta.h

```diff
@@ -18,4 +18,8 @@
 #include "x86_64/meta.h"
 #endif
 
+#ifdef MLK_SYS_PPC64LE
```

Comment:
@dannytsen This needs to be guarded by feature flags indicating supporting for the extensions used by the backend. Could you adjust this?